packages/cli/src/commands/versions/bump.test.ts

```diff
@@ -20,6 +20,8 @@ import { Command } from 'commander';
 import { resolve as resolvePath } from 'path';
 import { paths } from '../../lib/paths';
 import * as runObj from '../../lib/run';
+
+jest.mock('global-agent/bootstrap', () => {});
```

Comment:
Still works if you move this down to after the imports right? It should be hoisted anyway.

Could you add a comment explaining what's going on as well? 🙏